compiler/quilt/test/test_util.py

```diff
@@ -0,0 +1,31 @@
+"""
+Tests for quilt.tools.util
+"""
+
+import os
+
+from .utils import QuiltTestCase
+from ..tools.util import sub_dirs, sub_files
+
+class UtilTest(QuiltTestCase):
+    def test_sub_files(self):
+        mydir = os.path.dirname(__file__)
+        path = os.path.join(mydir, './listdir')
+
+        for f in sub_files(path):
+            assert os.path.isfile(os.path.join(path, f)), 'Expected only files'
```

Comment:
One more nitpick: this doesn't really test anything cause it's literally the implementation of the function it's testing. E.g., imagine that `isfile` returns `True` for both regular files and directories; the test won't catch that. What you want is just a hardcoded list, like `assert sub_files(path) == ['foo.txt']`, etc.

Reply:
@eode can you handle Dima's feedback above; he's right. the files are in `listdir`.

Reply:
@dimaryaz -- this more to your liking?

Reply:
Yup, was on it.